src/Microsoft.DotNet.Interactive/ChooseKeyValueStoreKernelDirective.cs

```diff
@@ -18,18 +18,18 @@ public class ChooseKeyValueStoreKernelDirective : ChooseKernelDirective
     {
         NameOption = new Option<string>(
             "--name",
-            "The name of the value to create. You can use #!share to retrieve this value from another subkernel.")
+            LocalizationResources.Magics_store_name_Description())
         {
             IsRequired = true
         };
 
         FromUrlOption = new Option<Uri>(
             "--from-url",
-            description: "Specifies a URL whose content will be stored.");
+            description: LocalizationResources.Magics_store_from_url_Description());
```

Comment:
```suggestion
            description: LocalizationResources.Magics_value_from_url_Description());
```

The word `value` is more appropriate here because the user-facing magic command for this kernel is `#!value`.

Reply:
changed to 'value'